test/weights.jl

```diff
@@ -85,8 +84,8 @@ w2 = rand(8)
 @test size(wsum(x, w1, 1)) == (1, 8)
 @test size(wsum(x, w2, 2)) == (6, 1)
 
-@test wsum(x, w1, 1) ≈ sum(x .* w1, 1)
-@test wsum(x, w2, 2) ≈ sum(x .* w2', 2)
+@test wsum(x, w1, 1) ≈ Compat.sum(x .* w1, dims = 1)
```

Comment:
Would `using Compat: sum` allow not repeating the prefix?

Reply:
It would have to be `import Compat: sum` in order to shadow the symbol from Base.

Reply:
Are you sure? `using` seems to be enough in my tests.

Reply:
Nope, I'm not sure. :) The behavior probably changed in 0.7 when Jeff made packages no longer load into Main by default.